adanet-init/adanet-init.ino

```diff
@@ -131,6 +134,36 @@ void setup()
   Serial.print("SCD4x Serial = ");
   Serial.println(scd4xSerial, HEX);
 
+  // configure temperature units for display
+  pref.begin("adanet-co2", false);
+  Serial.println("Select temperature display units [c]elsius [f]arenheit? [c]");
+  while (!Serial.available())
+    delay(10);
+
+  if (Serial.read() == 'f')
+  {
+    pref.putChar("temp_units", 'F');
+    Serial.println("Units set to Farenheit");
+  }
+  else
+  {
+    pref.putChar("temp_units", 'C');
+    Serial.println("Units set to Celsius");
+  }
+  pref.end();
+
+  Serial.println("Proceed to [c]alibration or [e]xit? [e]");
+  while (!Serial.available())
+    delay(10);
+
+  if (Serial.read() != 'c')
+  {
+    Serial.println("Done.");
+    delay(1000);
```

Comment:
Nit: do we need to `delay`?

Reply:
The delay is there to make sure the prefs lib has time to commit/write changes to flash in case of an early exit, no idea if it's needed but I'm sure it doesn't hurt, I'll add a comment.

Reply:
Perhaps should delay before printing `Done` then (in case someone yanks power)?